ui/app/AppLayouts/Communities/panels/OverviewSettingsFooter.qml

```diff
@@ -0,0 +1,141 @@
+import QtQuick 2.15
+import QtQml 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Control {
+    id: root
+    property bool isControlNode: true
+    property int loginType: Constants.LoginType.Password
+    property string communityName: ""
+
+    signal primaryButtonClicked
+    signal secondaryButtonClicked
+
+    QtObject {
+        id: d
+        
+        readonly property real verticalBreakPoint: 950
+        readonly property bool twoRowsLayout: contentItem.width <= verticalBreakPoint
+        readonly property string loginIcon: getLoginIcon()
+
+        property string paragraphTitle
+        property string paragraphSubtitle
+        property string primaryButtonText
+        property string primaryButtonIcon
+        property string secondaryButtonText
+        property string secondaryButtonIcon
+        property string indicatorBgColor
+        property string indicatorColor
+
+        function getLoginIcon() {
```

Comment:
Extract this function to some common place pls, this is like a second or third time I see this repeated somewhere :)

Reply:
Done. It's in `Constants`